output/tar/tar_output.go

```diff
@@ -51,10 +51,9 @@ func (i Output) Apply(rootPath string) <-chan error {
 
 		err = tar.Run()
 		if err != nil {
-			done <- err
+			done <- output.Report{output.UnknownError.Wrap(err).(*errors.Error), def.Output{}}
 			return
 		}
-
 	}()
 	return done
```

Comment:
I'm inclined to say that when errors are a struct member, you're expected to treat the rest with scepticism + care if the error is non-nil. So I agree with your choice in later commits, I think this is fine for now.